src/agent/src/config.rs

```diff
@@ -179,6 +182,7 @@ impl Default for AgentConfig {
             https_proxy: String::from(""),
             no_proxy: String::from(""),
             data_integrity: false,
+            enable_signature_verification: true,
```

Comment:
It seems like https://github.com/kata-containers/tests/issues/5228 is saying that the default should be false?

Reply:
Nope - the default is true (for security reasons), but we turn it off with the default kata-config for backwards compatibility

Reply:
I wouldn't exactly call this security critical, but I guess it's not too big of a deal either way.